apps/mobile/src/app/(home)/settings/display/index.tsx

```diff
@@ -28,46 +30,59 @@ export default function SettingsDisplayScreen() {
   const {
     accountDisplayPreference,
     bitcoinUnitPreference,
+    changePrivacyModePreference,
     fiatCurrencyPreference,
+    privacyModePreference,
     themePreference,
   } = useSettings();
   const { i18n } = useLingui();
 
+  function onUpdatePrivacyMode() {
+    changePrivacyModePreference(privacyModePreference === 'visible' ? 'hidden' : 'visible');
+  }
+
   return (
     <>
       <SettingsScreenLayout>
         <DisplayCell
           title={t`Theme`}
           caption={i18n._(capitalize(themePreference))}
           icon={<SunInCloudIcon />}
-          onCreateSheetRef={() => {
+          onPress={() => {
```

Comment:
I hope you don't mind me peeking in on some PRs - quick question about passing anon-functions - are we picking and choosing where to use `useCallback` based on performance concerns? These seem like good candidates to create via `useCallback` to ensure we're not needlessly re-rendering cells.

Reply:
We have had debates about overusing `useMemo` and `useCallback`:
https://kentcdodds.com/blog/usememo-and-usecallback

Not sure it is necessary on the cases you flagged?

Reply:
You're very likely right - probably safe (and wise) to not overuse it and add when necessary.